config/repos.yaml

```diff
@@ -176,6 +176,7 @@ Katello/foreman-gutterball:
 Katello/foreman_sam:
   redmine: katello
 Katello/hammer-cli-csv:
+  pr_scanner: true
```

Comment:
@domcleal I'm not sure what needs to be changed, but the hammer-cli-csv PRs are not getting linked properly to the redmine issue when submitted to GitHub. What do you think is missing/wrong with the config given that information?